_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -28,6 +28,21 @@ By default it is deployed using [systemd socket activation](http://0pointer.de/b
 ### Known Issues
 * The version of `rubygem-net-ssh` package required by Foreman is incompatible with the version provided EPEL8. Users who wish to install Foreman on EL8 should ensure that EPEL repository is disabled or that the conflicting package is excluded.
 * During upgrade testing, there have been a few cases where a 502 error was encountered when the installer tried to register the integrated Smart Proxy to Foreman. While we believe we have likely resolved the root cause, if this error is encountered, try re-running the installer again.
+* External authentication does not work properly with the Puma web server. This will be [fixed](https://projects.theforeman.org/issues/30535) in Foreman 2.2. In the mean time, users requiring external authentication should continue using the Passenger web server (which can be done by running the installer with the `--foreman-passenger` flag).
```

Comment:
```suggestion
* External authentication does not work properly with the Puma web server. This will be [fixed](https://projects.theforeman.org/issues/30535) in Foreman 2.2. In the meantime, users requiring external authentication should continue using the Passenger web server (which can be done by running the installer with the `--foreman-passenger` flag).
```